### PR TITLE
Make QMK Configurator work with e7-v1

### PR DESCRIPTION
**Pull request by masterzen**
This adds the missing info.json file for the e7-v1 keyboard which I
forgot to add to the original PR.
This allows to see the correct layout in QMK Configurator.

## Description

I forgot to add an `info.json` file for the E7-V1 keyboard in PR #4965. This makes configuring layouts with QMK Configurator harder than necessary.

This PR adds the needed `info.json` for all the supported layouts.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/e7v1/info.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "keyboard_name": "E7 - V1",
+    "url": "",
+    "maintainer": "masterzen",
+    "bootloader": "atmel-dfu",
```
Comment (mechmerlin) [inline keyboards/e7v1/info.json]: This tag actually isn't needed
```suggestion

```

File: keyboards/e7v1/info.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "keyboard_name": "E7 - V1",
```
Comment (mechmerlin) [inline keyboards/e7v1/info.json]: Looks funny with too many white spaces

```suggestion
    "keyboard_name": "E7-V1",
```

**Comment by masterzen** [comment]
@noroadsleft , you were right, the ISO layout was broken. It was missing a key as you found out, but also the PCB is strangely wired around the Enter key and this extra ISO key. The enter key is in column 13, but this extra key (which is placed on step before enter) is wired on column 14.

**Comment by masterzen** [comment]
I think everything is correct now and this is mergeable.

**Comment by noroadsleft** [comment]
> the PCB is strangely wired around the Enter key and this extra ISO key. The enter key is in column 13, but this extra key (which is placed on step before enter) is wired on column 14.

Lemme make sure I understand this correctly:

- ANSI Enter is 4th row down, 14th column across
- ISO Enter is 4th row down, **13th** column across
- ISO Hash (the 1u key) is 4th row down, 14th column across

Is that correct?

File: keyboards/e7v1/e7v1.h [hunk 3]
```diff
@@ -90,14 +90,14 @@
...
-	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313,             \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314,       \
 	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411,       K413, K414,       \
 	K500, K501, K502,             K505,                         K510,       K512, K513, K514, K515  \
 ) { \
 	{ K000,  K001,  K002,  K003,  KC_NO, K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014,  K015 }, \
 	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112, KC_NO,  K114,  K115 }, \
 	{ K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213, KC_NO,  K215 }, \
-	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  KC_NO, KC_NO }, \
+	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K314,  KC_313, KC_NO }, \
```
Comment (noroadsleft) [inline keyboards/e7v1/e7v1.h]:
```suggestion
	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K314,  K313, KC_NO }, \
```

**Comment by masterzen** [comment]
@noroadsleft 

It's not quite that, sorry if I was't clear. 

If starting to number everything from 1 (rows and columns), then:

* ANSI enter is on row 4, column 14
* ISO enter is also, row 4, column 14
* ISO hash/tilde is row 4, column 15, despite being physically before enter

File: keyboards/e7v1/e7v1.h (hunk 3 above)
Comment (masterzen) [reply keyboards/e7v1/e7v1.h]: oops, good catch!

File: keyboards/e7v1/e7v1.h [hunk 4]
```diff
@@ -90,14 +90,14 @@
 	K000, K001, K002, K003,       K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, \
 	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112,       K114, K115, \
 	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213,       K215, \
-	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313,             \
+	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314,       \
```
Comment (noroadsleft) [inline keyboards/e7v1/e7v1.h]:
```suggestion
	K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K214, K313,       \
```

File: keyboards/e7v1/e7v1.h [hunk 5]
```diff
@@ -90,14 +90,14 @@
...
 	K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411,       K413, K414,       \
 	K500, K501, K502,             K505,                         K510,       K512, K513, K514, K515  \
 ) { \
 	{ K000,  K001,  K002,  K003,  KC_NO, K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013,  K014,  K015 }, \
 	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  K112, KC_NO,  K114,  K115 }, \
-	{ K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213, KC_NO,  K215 }, \
-	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  KC_NO, KC_NO }, \
+	{ K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  K313,  K215 }, \
```
Comment (noroadsleft) [inline keyboards/e7v1/e7v1.h]:
```suggestion
	{ K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213,  K214,  K215 }, \
```

File: keyboards/e7v1/e7v1.h [hunk 6]
```diff
@@ -90,14 +90,14 @@
...
+	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K314,  KC_NO, KC_NO }, \
```
Comment (noroadsleft) [inline keyboards/e7v1/e7v1.h]:
```suggestion
	{ K300,  KC_NO, K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313,  KC_NO, KC_NO }, \
```

**Comment by masterzen** [comment]
@noroadsleft 

Many thanks for the help! And sorry for the mistakes. I now have a better understanding of the layouts.

I think it is now correct :)
But if it is not, do not hesitate to comment again :)

**Comment by noroadsleft** [comment]
@masterzen Thanks for being so patient and receptive. I worry about how contributors take my advice sometimes, especially when it's stuff that at a glance might be deemed insignificant.

**Comment by masterzen** [comment]
@noroadsleft no issues at all, I'm new to contributing to this project, and I want the things to be done correctly, so thanks for guiding me into the right direction :)

**Comment by drashna** [comment]
Just want to make sure, this is ready to merge, right?

**Comment by masterzen** [comment]
@mechmerlin thanks for merging this change!

I will work on regrouping the exclusive boards in their own directory this week :)

**Pull request merged**